components/Spotlight.js

```diff
@@ -1,17 +1,39 @@
 import Image from "next/image";
+import { styled } from "styled-components";
 
 export default function Spotlight({ pieces }) {
   const index = Math.floor(Math.random() * pieces.length);
   console.log(index);
   return (
-    <article key={pieces[index].slug}>
-      <Image
-        src={pieces[index].imageSource}
-        height={300}
-        width={200}
-        alt={pieces[index].name}
-      />
+    <StyledArticle key={pieces[index].slug}>
+      <ImageWrapper>
+        <StyledImage
+          src={pieces[index].imageSource}
+          height={300}
+          width={200}
+          alt={pieces[index].name}
+        />
+      </ImageWrapper>
       <h3>{pieces[index].artist}</h3>
-    </article>
+      <p>Just a famous Artist for you...</p>
+    </StyledArticle>
```

Comment:
Good solution! Another option could be that you move the logic to find a random art piece to your HomePage/index.js and just hand over the spotlightPiece to your SpotLight component as this otherwise does not need the information of all art pieces. :)

Reply:
thx, we will stick or now than to your solution.